ui-components/src/helpers/formatYesNoLabel.ts

```diff
@@ -1,9 +1,13 @@
 import { t } from "./translator"
 
-export const formatYesNoLabel = (value: boolean | null) => {
-  if (value === null || typeof value == "undefined") return t("t.n/a")
-  if (value) return t("t.yes")
-  if (!value) return t("t.no")
+export const formatYesNoLabel = (
+  value: boolean | null,
+  strings?: { yesString?: string; noString?: string; notApplicableString?: string }
+) => {
+  if (value === null || typeof value == "undefined")
+    return strings?.notApplicableString ?? t("t.n/a")
+  if (value) return strings?.yesString ?? t("t.yes")
+  if (!value) return strings?.noString ?? t("t.no")
 
   return t("t.n/a")
```

Comment:
This last return statement will never get hit since if the if statement on 9 fails the if statement on 10 will always pass.
So you could just do 
```
if (value) return strings?.yesString ?? t("t.yes")
return strings?.noString ?? t("t.no")

Reply:
Fixed!